### DISCUSSION TITLE
Can't get maxmind to work

### DISCUSSION THREAD

**Discussion by mbouclas** [plausible/analytics#1930] [Self-Hosted Support] [answered]
Followed the docs here [https://plausible.io/docs/self-hosting#1-maxmind-geolocation-database](url) but it doesn't seem to work. The map is never populated and the countries tab shows `No data yet`.
* Added the Maxmind credentials in the `geoip/geoip.conf`
* Added the conf variable `GEOLITE2_COUNTRY_DB=/geoip/GeoLite2-Country.mmdb`
* started the stack using `docker-compose -f docker-compose.yml -f geoip/docker-compose.geoip.yml up -d`
* The `GeoLite2-Country.mmdb` file is identical on both containers
* checked the docker logs for any possible errors, but nothing there
* Cloudflare is the DNS

Any ideas?

**Comment by ghost** [comment]
Have you mounted the database in the docker image?

Easier is to create a key in Maxmind and let plausible download the database and mount it.

**Comment by mbouclas** [comment]
That's what i did. I have set the key in the `geoip.conf`, the db was downloaded successfully in the container and i can see the file by doing an `exec ls` in that container and in the `/geoip` folder.

**Comment by ghost** [comment]
When I tested it I did not indicate the path to the database, only the Maxmind ID and license key (free version).

Does it work with the default database?

**Comment by mbouclas** [comment]
no, that's why i went for Maxmind

**Comment by ghost** [comment]
Do you use nginx as a proxy?

    location / {
        proxy_pass http://127.0.0.1:8000;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }

**Comment by mbouclas** [comment, marked as answer]
ah, was missing the `proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;` part on the config.
Thanks so much

**Comment by DenjixPochita** [comment]
MaxMind? You don't need it, as Plausible uses db-ip. And if you are discussion about city lvl data, it's not yet available in self-hosted version yet https://github.com/plausible/analytics/discussions/1601#discussioncomment-1952442.
PS: Let me know if I am wrong!

**Reply by codebude** [reply]
@DenjixPochita thanks for pointing that out. I thought something is broken with my self-hosted instance, because city-level data is missing...

**Reply by ruslandoga** [reply]
👋 @codebude 

City-level geolocation is available as of [`v1.5.1`](https://github.com/plausible/analytics/discussions/2507) and in [`v2.0.0-rc.0`](https://github.com/plausible/analytics/discussions/2896) DB management has been automated.
